tests/neural_compressor/test_optimization.py

```diff
@@ -142,6 +142,37 @@ def test_static_quantization(self):
             # Verification quantized model was correctly loaded
             self.assertEqual(q_model_result, loaded_model_result)
 
+    def test_static_quantization_with_calib_func(self):
+        model_name = "distilbert-base-uncased-finetuned-sst-2-english"
+        config_path = os.path.join(os.path.dirname(os.path.abspath(__file__)))
+
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            model, trainer, eval_func = self.helper(model_name, tmp_dir)
+            model.config.save_pretrained(tmp_dir)
+            model_result = eval_func(model)
+            q8_config = IncQuantizationConfig.from_pretrained(config_path)
+            q8_config.set_config("quantization.approach", IncQuantizationMode.STATIC.value)
+            q8_config.set_config("tuning.accuracy_criterion.relative", 0.04)
+            q8_config.set_config("model.framework", "pytorch_fx")
+            quantizer = IncQuantizer(
+                q8_config, eval_func=eval_func, calib_dataloader=trainer.get_eval_dataloader(), calib_func=eval_func
```

Comment:
For what kind of model is a `calib_func` required for post-training static quantization ? Is there other models than diffusion models which would need it ?

Reply:
I think calib_func is applicable to all pipelines containing multiple models. In these pipeline, if user want to quantize one model, calib_func will be used.